src/Event/SmsEvents.php

```diff
@@ -27,7 +27,7 @@
   const MESSAGE_GATEWAY = 'sms.message.gateway';
 
   /**
-   * Process and chunk a SMS message before it is queued, sent, or received.
+   * Process a SMS message before it is queued, sent, or received.
```

Comment:
The Symfony* classes get straight to the point and describes when the event is dispatched. 

The Commerce ecosystem has adopted a standard of prefixing its events with "Name of the event fired". However I don't necessarily care for it.

* Examples: `\Symfony\Component\Console\ConsoleEvents`, `\Symfony\Component\HttpKernel\KernelEvents`